forem/forem
# Add UI for member index v2 actions dropdown buttons

## What type of PR is this? (check all applicable)


## Description

This PR adds the new buttons to the user actions overflow menu when the `member_index_view` feature flag is enabled.

The associated actions are covered in separate issues, so for the time being clicking them won't do anything 🙂 

## Related Tickets & Documents



- Closes https://github.com/forem/forem/issues/17478

## QA Instructions, Screenshots, Recordings

With the `member_index_view` feature flag **disabled**:
- visit `/admin/member_manager/users`
- check that when you click the overflow menu, you only see the "Copy email" button, and it continues to work as normal

With the `member_index_view` feature flag **enabled**:
- visit `/admin/member_manager/users`
- click the overflow menu, and check you see the additional buttons as per [the design](https://www.figma.com/file/Ejc7VELuN00n1SZrkUTBcQ/Member-List-View?node-id=1124%3A5686)


![expanded dropdown menu with buttons](https://user-images.githubusercontent.com/20773163/169816223-04a7f0fa-9a92-4418-aa3e-0c66e527eca6.png)


### UI accessibility concerns?

Not much - since we now have several buttons, I've wrapped them in a list to help with readability/navigation

## Added/updated tests?

- [X] No, and this is why: We'll add them when we attach the functionality

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: just an initial step towards the feature
